packages/database/src/core/Repo.ts

```diff
@@ -296,6 +297,42 @@ export class Repo {
     return this.nextWriteId_++;
   }
 
+  get(query: Query): Promise<DataSnapshot> {
+    return this.server_.get(query).then(
+      payload => {
+        const node = nodeFromJSON(payload as string);
+        const events = this.serverSyncTree_.applyServerOverwrite(
+          query.path,
+          node
+        );
+        this.eventQueue_.raiseEventsAtPath(query.path, events);
+        return Promise.resolve(
+          new DataSnapshot(
+            node,
+            query.getRef(),
+            query.getQueryParams().getIndex()
+          )
+        );
+      },
+      err => {
```

Comment:
Do we log the `err` anywhere? I assume this will be in the WebChannel logs, but if not, we may want to log it here as well.